htop.c

```diff
@@ -35,7 +35,8 @@ static void printVersionFlag(void) {
 
 static void printHelpFlag(void) {
    fputs("htop " VERSION "\n"
-         "Released under the GNU GPL.\n\n"
+	 COPYRIGHT "\n"
```

Comment:
Is this a tab?

Reply:
Hehe, I fear yes. Old habbits...

Reply:
Should be fixed, thanks for the nudge @cgzones